Sources/Extensions/CustomTypes/AnalyticsService+GDS.swift

```diff
@@ -23,9 +24,18 @@ extension AnalyticsService {
     }
     
     mutating func setAdditionalParameters(appTaxonomy: AppTaxonomy) {
+        var taxonomyLevel3: String {
```

Comment:
This is a real [open-closed principle](https://en.wikipedia.org/wiki/Open–closed_principle) violation for now, but I think we can let it through given we're going to address it with the Logging package change: https://govukverify.atlassian.net/browse/DCMAW-10246